reference/components/cf-monitord.markdown

```diff
@@ -120,6 +120,30 @@ into agent variables in the `$(mon.`name`)` context.
 
 Note: There is no way for force a refresh of the monitored data.
 
+## Statistical Classes
+
+`cf-monitord` automatically defines classes based on the observation of the data
+is has collected. Classes defined are named for the measurement id (the promise
+handle in the case of custom measurement promises) with prefixes and or suffixes
+depending on the measurement.
+
+The following suffixes may be used when defining classes:
+
+* `_high` :: The last measurement was greater than the average.
+* `_low` :: The last measurement was lower than the average.
+* `_normal` ::
+* `_ldt` ::
+* `_dev1` :: The last measurement was at least 1 standard deviation higher than the average.
+* `_dev2` :: The last measurement was at least 2 standard deviations higher than the average. These classes are persistently defined for a number of minutes.
+* `_anomaly` :: The last measurement was at least 3 standard deviations than the average. These classes are persistently defined for a number of minutes.
+* `_microanomaly` :: The last measurement was at least 2 standard deviations higher than the average.
```

Comment:
```suggestion
* `_microanomaly` :: A deviation of more than 2 standard deviations was detected, but the difference between the last measurement and the average was below the noise threshold. (So it could mean that this anomalous value is just due to noise).
```